tests/products-page.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test'
+
+test('products page should contain an image in the link', async ({ page }) => {
+  await page.goto(`/products`)
+  const link = await page.getByRole('link', {
+    name: 'Forzudo Blanco Forzudo Blanco Bierzo, Leon, Nordvest Spanien 100 cl. / 15%',
```

Comment:
is this needed?

Reply:
No, there are other ways to get to the link. now I see how